vhdbuilder/packer/install-dependencies.sh

```diff
@@ -504,6 +504,25 @@ EOF
 
 fi
 
+if grep -q "GB200" <<< "$FEATURE_FLAGS"; then
+  # The GB200 feature flag should only be set for arm64 and Ubuntu 24.04, but validate
+  if [ ${UBUNTU_RELEASE} = "24.04" ] && [ ${CPU_ARCH} = "arm64" ]; then
+    # The open series driver is required for the GB200 platform. Dmesg output
+    # will appear directing the reader away from the proprietary driver. The GPUs
+    # are also not visible in nvidia-smi output with the proprietary drivers
+    apt install -y \
+      mlnx-ofed-kernel-dkms \
+      mlnx-ofed-kernel-utils \
+      mlnx-ofed-basic \
+      rdma-core \
+      ibverbs-utils \
+      ibverbs-providers
+
+    systemctl restart openibd
+    ofed_info -s
```

Comment:
is the idea that this gets stored to the cluster-provisionning logs ?  we shoudl redirect the output the file is we want to store the output.

Reply:
the intention here is just to help debug driver installation issue if any